frontend/src/components/IstioWizards/GatewaySelector.tsx

```diff
@@ -181,7 +182,7 @@ export class GatewaySelector extends React.Component<Props, GatewaySelectorState
                 id="includeMesh"
                 label={
                   <>
-                    Include <b>mesh</b> gateway
+                  {t('Include')} <b>mesh</b> {t('gateway')}
```

Comment:
In this case, I think it is better to have the entire statement translated rather than just the bold word `mesh`. 
```suggestion
                  {t('Include mesh gateway')}
```